QuizWebsite/src/problem/Abstract/AbstractAnswer.java

```diff
@@ -0,0 +1,67 @@
+package problem.Abstract;
+
+public abstract class AbstractAnswer {
+	private int grade;
+	private boolean graded = false;
+	private int testId;
+	private Object userAnswer;
+
+
+	/**
+	 *  Constructor stores test's id number and user's answer object.
+	 */
+	public AbstractAnswer(int newTestId, Object newUserAnswer) 
+	{
+		testId = newTestId;
+		userAnswer = newUserAnswer;
+	}
+
+	/**
+	 * Returns an object with all information about Answer.
+	 * @return answer.
+	 */
+	public Object getAnswer()
+	{	
+		return userAnswer;
+	}
+
+	/**
+	 * Sets the grade user in object.
+	 */
+	public void setGrade(int newGrade)

```

Comment:
Some answers depend on each other and cant be checked independetly. This is an overall Grade